share/goodie/cheat_sheets/json/win-run.json

```diff
@@ -0,0 +1,115 @@
+{
+  "id": "windows_run_cheat_sheet",
```

Comment:
@mahi30795 windows_run_command_cheat_sheet would be better.
Also for file name make the same correction